packages/react/src/side-nav/SideNav.tsx

```diff
@@ -64,7 +64,7 @@ export const SideNav = forwardRef<HTMLElement, SideNavProps>(
             transition={animations ? "all" : undefined}
             w={expanded ? "224" : "56"}
           >
-            <nav aria-label="Global Navigation" ref={ref}>
+            <nav aria-label="side" ref={ref}>
```

Comment:
let's do Main Navigation here
```suggestion
            <nav aria-label="Main Navigation" ref={ref}>
```

just `side` by itself won't be very meaningful

another alternative could be `Side navigation`

Reply:
Vai, the navigation role would trigger the screen reader side + navigation 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/navigation_role#best_practices